docs/configuration/config-file-reference.md

```diff
@@ -858,6 +858,12 @@ store_gateway_client:
 # (ingesters shuffle sharding on read path is disabled).
 # CLI flag: -querier.shuffle-sharding-ingesters-lookback-period
 [shuffle_sharding_ingesters_lookback_period: <duration> | default = 0s]
+
+tenant_federation:
+  # If enabled, multi tenant query federation can be used by supplying multiple
+  # tenant IDs in the read path (experimental).
+  # CLI flag: -querier.tenant-federation.enabled
+  [enabled: <boolean> | default = false]
```

Comment:
If some cluster enables multi tenant querying, the flag should be set across all the instances and not only on the query frontend/scheduler, so that e.g. the distributor rejects multi tenant ingestion. I feel the prefix `-querier.` is misleading

Reply:
I have moved the whole implementation into it's own package with it's own config. This should be making it much clearer. 

The wrapping of the queryable now happens in modules.go so it should be more extensible...

Reply:
👍